Assets/Scripts/Level/TutorialTriggerGroup.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using UnityEngine;
+
+namespace Assets.Scripts.Level
+{
+    public class TutorialTriggerGroup : MonoBehaviour
+    {
+
+        public TutorialTrigger[] triggers;
+        public InGameUI gameUI;

```

Comment:
you could populate both this with FindObjectOfType/FindObjectsOfType in Start() so that we don't have to manually set it up in every scene. The minor performance cost is worth the saving of our time

Reply:
If you're sure! I can't find a good way to grab InGameUI without doing the FindObjectOfType() so I chose to handle it manually, but I'll do so if that's alright.